scipy/optimize/_basinhopping.py

```diff
@@ -452,7 +452,7 @@ def basinhopping(func, x0, niter=100, T=1.0, stepsize=0.5,
     Returns
     -------
     res : OptimizeResult
-        The optimization result represented as a ``OptimizeResult`` object.
+        The optimization result represented as a `OptimizeResult` object.
```

Comment:
@tupui we should remember to check whether this renders correctly as a link (same as `basinhopping` above). But in the meantime I'm merging this as an improvement.